pageserver/src/tenant/timeline/delete.rs

```diff
@@ -334,6 +336,7 @@ impl DeleteTimelineFlow {
         tenant: &Tenant,
         timeline_id: TimelineId,
         allow_offloaded_children: bool,
+        set_stopping: bool,
     ) -> Result<(TimelineOrOffloaded, DeletionGuard), DeleteTimelineError> {
```

Comment:
There are only two users of `prepare` , i.e., two sets of values for the bool args.
But reading this function in isolation, I have to reason about all four possible args.

Would prefer an enum, makes it simpler to reason about what we do.
Can be a follow-up though.

Reply:
:+1: will put it into a followup.